packages/app/components/EventCard/EventCard.js

```diff
@@ -0,0 +1,51 @@
+import React, { useCallback, useEffect, useState} from 'react';
```

Comment:
Your code editor will highlight which imported variables are not being used. I suggest you remove the ones you're not using.